tardis/adapters/sites/moab.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, machine_type: str, site_name: str):
     async def deploy_resource(
         self, resource_attributes: AttributeDict
     ) -> AttributeDict:
-        machine_configuration = self.configuration.MachineTypeConfiguration[
+        machine_configuration = self._configuration.MachineTypeConfiguration[
             self._machine_type
         ]
```

Comment:
What is the idea to not include the ``machine_configuration`` property in the base class?

Reply:
It is only accessed in the implementation of the `SiteAdapter`. No other part of the code is relying on its existence. 

You think I should make it more uniform?